pom.xml

```diff
@@ -62,7 +62,7 @@
   <properties>
     <revision>1.6</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.414.3</jenkins.version>
+    <jenkins.version>2.440</jenkins.version>
```

Comment:
This (and the BOM line switch) are not exactly necessary and could be reverted, though I think it makes it clearer for purposes of tests and `hpi:run` to run against the core versions which lack these fields in `JNLPLauncher` and include an `agent.jar` with the rewritten CLI options.